model/task/task.go

```diff
@@ -82,18 +82,17 @@ type Task struct {
 	DependenciesMetTime    time.Time `bson:"dependencies_met_time,omitempty" json:"dependencies_met_time,omitempty"`
 	ContainerAllocatedTime time.Time `bson:"container_allocated_time,omitempty" json:"container_allocated_time,omitempty"`
 
-	Version           string              `bson:"version" json:"version,omitempty"`
-	Project           string              `bson:"branch" json:"branch,omitempty"`
-	Revision          string              `bson:"gitspec" json:"gitspec"`
-	Priority          int64               `bson:"priority" json:"priority"`
-	TaskGroup         string              `bson:"task_group" json:"task_group"`
-	TaskGroupMaxHosts int                 `bson:"task_group_max_hosts,omitempty" json:"task_group_max_hosts,omitempty"`
-	TaskGroupOrder    int                 `bson:"task_group_order,omitempty" json:"task_group_order,omitempty"`
-	Logs              *apimodels.TaskLogs `bson:"logs,omitempty" json:"logs,omitempty"`
```

Comment:
Should we remove [the type](https://github.com/evergreen-ci/evergreen/blob/9fd0768efec624b4c77318fa07b3551c1529f2bb/apimodels/agent_models.go#L77-L81) too?

Reply:
🤔 I can't find where it's used.
Could you link the specific line?

Reply:
🤔 I can't find where it's used.
Could you link the specific line?

Reply:
The evergreen sender is used [here](https://github.com/evergreen-ci/evergreen/blob/main/agent/internal/client/base_client.go#L478). We only use it for local dev, I have [EVG-17662](https://jira.mongodb.org/browse/EVG-17662) to replace it. The `apimodels.TaskLogs` type is used by the agent communicator's [SendLogMessages implementation](https://github.com/evergreen-ci/evergreen/blob/main/agent/internal/client/base_client.go#L492) used, in turn, by the evergreen log sender [here](https://github.com/evergreen-ci/evergreen/blob/main/agent/internal/client/evergreen_sender.go#L76).

Reply:
LOL we have something called `TaskLogs` and another thing `TaskLog`. My bad.

Reply:
Done. Good catch!

Reply:
thank you! Sorry to belabor the point.